Tests/appstoreconnect-cliTests/Operations/GetBetaTesterInfoOperationTests.swift

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+import XCTest
+
+final class GetBetaTesterInfoOperationTests: XCTestCase {
+    typealias Dependencies = GetBetaTesterInfoOperation.Dependencies
+
+    let options = GetBetaTesterInfoOptions(id: "abc")
+
+    func testExecute_success() {
+        let dependencies: Dependencies = .createdSuccess
+
+        let operation = GetBetaTesterInfoOperation(options: options)
+
+        let result = Result {
+            try operation.execute(with: dependencies).await()
+        }
+
+        switch result {
+            case .success(let betaTester):
+                XCTAssertEqual(betaTester.betaGroups?[0], "FooGroup")
+                XCTAssertEqual(betaTester.apps?[0], "com.example.fooapp")
+                XCTAssertEqual(betaTester.apps?[1], "com.example.fooapp")
+                XCTAssertEqual(betaTester.firstName, "Foo")
+            default:
+                XCTFail("Error happened when parsing get beta tester response")
+        }
+    }
+
+    func testExecute_propagatesUpstreamErrors() {
+        let dependencies: Dependencies = .createdFailed
+
+        let operation = GetBetaTesterInfoOperation(options: options)
+
+        let result = Result {
+            try operation.execute(with: dependencies).await()
+        }
+
+        let expectedError = TestError.somethingBadHappened
+
+        switch result {
+            case .failure(let error as TestError):
+                XCTAssertEqual(expectedError, error)
+            default:
+                XCTFail("Expected failure with: \(expectedError), got: \(result)")
+        }
+    }
+}
+
+private extension GetBetaTesterInfoOperationTests.Dependencies {
+
+    static let createdSuccessResponse = """
```

Comment:
I'm not a fan of large chunks of JSON in tests.  You should copy the approach I took in the ASC SDK here: https://github.com/ittybittyapps/appstoreconnect-swift-sdk/blob/fix/profile-parsing/Tests/Models/Responses/ProfilesResponseTests.swift

Reply:
Ie, create some infrastructure around loading "fixtures" from files.

Reply:
I have raised this for this work #126